system/lib/webgpu/webgpu_cpp.cpp

```diff
@@ -1,6 +1,9 @@
+
```

Comment:
Missing copyright header?

Reply:
Huh, this needs to be fixed upstream. In the meantime let me add the header we have in the template since this one is more important.

Reply:
if it really is from 2017 then this is fine.. I didn't realize webgpu went back that far :)

Reply:
yup, and this the enum class bitmasks utility probably hasn't even changed much since then. tbd if I update that.

The generated headers will have 2024 in them because the date is written at generation time, so I put those in as 2024.